recipes/imp/patch_protoc.py

```diff
@@ -0,0 +1,29 @@
+"""Patch the protoc-generated header to resolve export symbols"""
```

Comment:
Where is this file used?

Reply:
It is a replacement for upstream's script, which didn't work on Windows: https://github.com/salilab/npctransport/

Although I notice that we've actually pulled in those changes upstream now, so this should no longer be necessary. Thanks for bringing this to my attention!